README.md

```diff
@@ -58,3 +58,22 @@ This function will 'save' all the strings sent to it in any format, be it a list
 **Notes**:
 - When `typeof` of an input evaluates to `object` or `undefined`, this argument will be ignored. This happens to any kind of object, map, null or undefined values. To know more about the `typeof` operand, follow this link: [typeof - MDN](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/typeof)
 - While arrow functions may be more clean and concise, they do not bind a context to the execution of the function (`this`), so in case you need to use the helper inside a function that is going to be passed as an argument, write an old fashion function. More info on arrow functions: [Arrow Functions - MDN](https://developer.mozilla.org/en/docs/Web/JavaScript/Reference/Functions/Arrow_functions)
+
+### catIf(..args, condition)
+This function will 'save' all the strings sent to it in any format, be it a list of strings, an array of strings, a function even can be sent and it will be executed, saving the result for further processes.
```

Comment:
What you mean with -> *a function even can be sent and it will be execute, saving the result for further processes*?